plugins/PagePerformance/lang/en.json

```diff
@@ -0,0 +1,34 @@
+{
+    "PagePerformance": {
+        "ColumnAveragePageLoadTime": "Avg. page load time",
+        "ColumnAveragePageLoadTimeDocumentation": "Average time how long it takes from requesting a page until the page is fully rendered within the browser",
```

Comment:
btw not sure I'm seeing it... is there a metric giving us the server time? Meaning eg the time between sending the last byte of the request and receiving the first byte of the response. AKA how long did the server take to start sending a response. This be very valuable.

Reply:
also for all the metrics be good to write eg `Average time in seconds` or `Average time in ms` just in case someone fetches eg the unformatted metrics through the API etc.